### DISCUSSION TITLE
core version change DOM after toolbar

### DISCUSSION THREAD

**Discussion by teafmartins** [serenity-is/Serenity#7173] [Help] [answered]
Hello,

How can do this MVC code in new Serenity Core version?

```
 constructor() {
      super();
      
      $('<div id="InfoN">img src="/images/teste.png"></div>')
          .insertAfter(this.toolbar.element);
}
```

Thanks

**Comment by VictorTomaili** [comment, marked as answer]
this should work. 

```ts
Fluent("div").attr("id", "infoN").append(
    Fluent("img").attr("src", resolveUrl("~/Content/Images/test.png"))
).insertAfter(this.toolbar.element);
```

You can also use the vanilla code like `document.createElement("div")...` instead jquery. 
The other way is using tsx and jsx-dom. 

Check the other code samples 
https://github.com/serenity-is/common-features

**Comment by teafmartins** [comment]
@VictorTomaili , 
As usual with your ideas, it worked like a charm.
Thank You
